rbac/management/workspace/model.py

```diff
@@ -64,10 +64,23 @@ def clean(self):
         if self.type == self.Types.ROOT:
             if self.parent is not None:
                 raise ValidationError({"root_parent": ("Root workspace must not have a parent.")})
-        elif self.parent_id is None:
-            raise ValidationError({"parent_id": ("This field cannot be blank for non-root type workspaces.")})
-        elif self.type == self.Types.DEFAULT and self.parent.type != self.Types.ROOT:
-            raise ValidationError({"default_parent": ("Default workspace must have a root parent.")})
+
+        elif self.type == self.Types.DEFAULT:
+            if self.parent is None:
+                raise ValidationError({"default_parent": ("Default workspace must have a parent workspace.")})
+            if self.parent.type != self.Types.ROOT:
+                raise ValidationError({"default_parent": ("Default workspace must have a root parent.")})
+        elif self.type == self.Types.STANDARD:
+            if self.parent_id is None:
+                workspace_object = Workspace.objects.get(type=self.Types.DEFAULT, tenant=self.tenant_id)
+                self.parent = workspace_object
+                self.parent_id = workspace_object.id
+
+        elif self.type == self.Types.UNGROUPED_HOSTS:
```

Comment:
Maybe we can put the self.type == self.Types.UNGROUPED_HOSTS and self.type == self.Types.DEFAULT in else. then raise ValidationError(
                    {"workspace": (f"{self.type} workspaces must have a parent workspace.")}
                )